torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -60,7 +60,7 @@ XlaOpVector SizeNode::Lower(LoweringContext* loctx) const {
   return ReturnOp(xla::GetDimensionSize(input, this->dim_), loctx);
 }
 
-std::string SizeNode::ToString() const { return "SizeNode"; }
+std::string SizeNode::ToString() const { return "SizeNode for op aten::size"; }
```

Comment:
this one I guess you can just use `aten::size`

Reply:
done